### PR TITLE
handle parent account id being returned when requesting account

### PR DESCRIPTION
**Pull request by Wozacosta**
JIRA issue: https://ledgerhq.atlassian.net/jira/software/c/projects/LIVE/boards/623?selectedIssue=LIVE-11197
Ledger Live pull request : https://github.com/LedgerHQ/ledger-live/pull/6157

### CODE REVIEW

File: packages/server/src/internalHandlers/account.ts [hunk 1]
```diff
@@ -65,6 +65,7 @@ export const request: RPCHandler<AccountRequest["result"]> = async (
 
   return {
     rawAccount: serializeAccount(account),
+    parentAccountId: account.parentAccountId,
```
Comment (Justkant) [inline packages/server/src/internalHandlers/account.ts:68]: You should probably update `serializeAccount`

**Review by Justkant — COMMENTED** [review]
Probably something to update here too

File: packages/server/src/internalHandlers/account.ts (hunk 1 above)
Comment (Wozacosta) [reply packages/server/src/internalHandlers/account.ts]: Done

File: packages/core/src/accounts/serializers.ts [hunk 2]
```diff
@@ -28,6 +29,7 @@ export function serializeAccount({
     spendableBalance: spendableBalance.toString(),
     blockHeight,
     lastSyncDate: lastSyncDate.toISOString(),
+    parentAccountId,
```
Comment (Justkant) [inline packages/core/src/accounts/serializers.ts:32]: I think you also need the same in the `deserializeAccount` function

**Review by Justkant — COMMENTED** [review]
We might also need the field in the `deserializeAccount`

File: packages/core/src/accounts/serializers.ts (hunk 2 above)
Comment (Wozacosta) [reply packages/core/src/accounts/serializers.ts:32]: Done, good catch

File: .changeset/brave-icons-repair.md [hunk 3]
```diff
@@ -0,0 +1,6 @@
+---
+"@ledgerhq/wallet-api-server": patch
```
Comment (Justkant) [inline .changeset/brave-icons-repair.md:2]: You're only modifying the core
```suggestion
```

Comment (Wozacosta) [reply .changeset/brave-icons-repair.md]: Cleaned up 🧹

**Review by Justkant — APPROVED** [review]
LGTM

**Pull request merged**